packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -94,6 +94,12 @@ export class Dropdown {
    */
   @Prop() header?: string;
 
+  /**
+   * If set to true an open dropdown will close if the trigger element gets clicked again.
+   * @since 2.1.0
+   */
+  @Prop() triggerToggles: boolean;
```

Comment:
Should be part of the close behavior imo, there should be no additonal property.